suites/pacific/cephadm/tier-1_5-2_new-feature.yaml

```diff
@@ -0,0 +1,142 @@
+#===============================================================================================
+#-------------------------------------
+#----- Ceph code changes in 5.2 ------
+#--------------------------------------
+# * NFS deployment with pool and namespace is not supported.
+#
+#--------------------------------------
+#
+# Tier-1 test cases for
+#    - Bootstrap
+#    - Host management
+#      - Add nodes
+#      - SetAddress
+#      - Attach node labels
+#    - Apply services - mon, mgr and raw mode OSDs
+#    - Ceph cluster health check
+#===============================================================================================
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: Cephadm Bootstrap with custom dashboard credentials
+      desc: Bootstrap with initial-dashboard user and password option
+      module: test_bootstrap.py
+      polarion-id: CEPH-83573719
```

Comment:
If this bootstrap test is already part of some other test suite, i think we need not add the polarion-id here for it. Bootstrap can just be another setup step like install_prereq if we're not doing any new way of bootstrapping specific to this test suite.